vhdbuilder/packer/install-dependencies.sh

```diff
@@ -443,6 +444,22 @@ for KUBERNETES_VERSION in ${PATCHED_HYPERKUBE_IMAGES}; do
   CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/hyperkube:v${KUBERNETES_VERSION}"
   pullContainerImage "docker" ${CONTAINER_IMAGE}
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+
+  if (($(echo ${KUBERNETES_VERSION} | cut -d"." -f2) > 18)); then
```

Comment:
does this ever happen? Now the highest version is 1.18.x so this would never run

Reply:
This is preparing for 1.19, but I agree we should have one 1.19 for verification purpose